gcn_monitor/cli.py

```diff
@@ -51,7 +52,7 @@ def main(prometheus, loglevel):
     * Convert to upper-case.
     * Replace periods (`.`) with single underscores (`_`).
     * Replace dashes (`-`) with double underscores (`__`).
-    * Replace underscores (`-`) with triple underscores (`___`).
+    * Replace underscores (`_`) with triple underscores (`___`).
```

Comment:
This typo is probably repeated across all of these Docker/Python projects. Please open separate PRs to fix the typos (or, in this one instance, push the typo fix directly to main).

Reply:
Ah okay, I will just do it as a separate PR and rebase this one